### ISSUE TITLE
Member List Synchronization 

### ISSUE THREAD

**Issue by psytron** [EffectiveAF/effective#42]
Sometimes the members list falls out of sync and some members may not show up: 
Screenshot: 
![Screen Shot 2022-03-18 at 1 06 44 PM](https://user-images.githubusercontent.com/7834453/159076355-661414e3-3a59-48f0-8f09-99789ef102db.png)

**Comment by elimisteve** [comment]
Yup, there's a bug where the member list doesn't update in real-time like everything else.

Thanks!

**Comment by psytron** [comment]
I know I already mentioned this one but was just tagging to keep it organized in the appropriate stream
